### ISSUE TITLE
UPDATE with Named Parameter set to a list containing booleans results in _NOT_PARSED_

### ISSUE THREAD

**Issue by mmacfadden** [orientechnologies/orientdb#4776]
There is a an issue with SQL named parameters in 2.1, when the value of the named parameter is a list containing booleans.

We start out with a doc that looks like this:
```javascript
{
    "@type":"d",
    "@rid":"#9:0",
    "@version":1,
    "@class":"test",
    "id":1,
    "boolean":false,
    "integerList":[],
    "booleanList":[]
}
```

We then use this updated statement:

```sql
UPDATE test SET boolean = :boolean, booleanList = :booleanList, integerList = :integerList WHERE id = 1
```
Then execute that with the following named parameters:

:boolean = true
:booleanList = [true]
:integerList = [1]

This results in the following document:

```javascript
{
    "@type":"d",
    "@rid":"#9:0",
    "@version":2,
    "@class":"test",
    "id":1,
    "boolean":true,
    "integerList":[1],
    "booleanList":["_NOT_PARSED_"]
}

Interestingly, setting the named parameters to a boolean works.  Setting a named parameter to a list of integers works.  Setting the named parameter to a list of booleans results in "_NOT_PARSED_".

I have created a gist that reproduces the issue: 

https://gist.github.com/mmacfadden/cd4f432af79a5626dd94

**Comment by luigidellaquila** [comment]
fixed in 2.1.x branch, porting to develop.
The fix will be released with v 2.1.1 hotfix

**Comment by mmacfadden** [comment]
Thanks guys!
